README.md

```diff
@@ -22,3 +39,34 @@ Please read the [`Adding Custom Modules`](docs/ADDING_CUSTOM_MODULE.md) section
 ## Attribution
 
 This project uses [`DevUtils`](https://devutils.com) as inspiration.
+
+## 🤝 Contributing
+
+Contributions to EthUnwrapped are welcome! If you would like to contribute, please follow these steps:
```

Comment:
Please change `EthUnwrapped` to `Devty`

Reply:
Also can you please mention about the `docs` folder under this section? So if someone wants to add a custom module they can look into it

Reply:
Also please remove the `How can I add my custom module?` section since that can be merged with this section